run

```diff
@@ -565,6 +565,9 @@ build_desc() {
   echo "   --buildpath <build_directory>        : Change the build path."
   echo "                                          Default: build"
   echo "                                          Associated environment variable: CMAKE_BUILD_PATH"
+  echo "   --benchmark                          : Build for Holoscan Flow Benchmarking."
+  echo "                                          Make sure the application is patched before"
+  echo "                                          building with this option."
```

Comment:
Could we automate this step to reduce dev responsibilities? Can we run `patch_application.sh` automatically prior to the CMake build when `--benchmark` is specified?

Reply:
Thanks for the suggestion! Automated it.